gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2098,16 +2099,26 @@ public GoogleCloudStorageItemInfo composeObjects(
 
   <RequestT extends StorageRequest<?>> RequestT initializeRequest(
       RequestT request, String bucketName) throws IOException {
+    String accessToken = null;
     if (storageRequestAuthorizer != null) {
-      storageRequestAuthorizer.authorize(request);
+      try {
+        accessToken = storageRequestAuthorizer.authorize(request);
+      } catch (UnsupportedOperationException e) {
+        // For not supported methods, do nothing.
```

Comment:
nit: unsupported

Reply:
Can we log a warn statement instead of ignoring the exception ?